lib/shenzhen/commands/build.rb

```diff
@@ -150,7 +150,8 @@
 
   def validate_xcode_version!
     version = Shenzhen::XcodeBuild.version
-    say_error "Shenzhen requires Xcode 4 (found #{version}). Please install or switch to the latest Xcode." and abort if version < "4.0.0"
+    majorVersion = version.split('.')[0].to_i
```

Comment:
Can you fix the naming? It must be consistent with the project.

Reply:
Doh, sure thing.

Reply:
I've made the requested change. Thanks for your feedback!